crates/matrix-sdk-crypto/src/types/events/olm_v1.rs

```diff
@@ -177,7 +177,7 @@ where
     pub recipient_keys: OlmV1Keys,
     /// The device keys if supplied as per MSC4147
     #[serde(rename = "org.matrix.msc4147.device_keys")]
```

Comment:
Yep, unless I'm missing something, this is only renaming the variable when it's used on the Rust side, so this is an internal renaming, so this may not be what you intended to do?

Reply:
Err, it's renaming the field during serialization/deserialization. The serialized JSON will contain `org.matrix.msc4147.device_keys` during serialization and we will expect `org.matrix.msc4147.device_keys` during deserialization. While the code have the more pleasant name.

Reply:
Well spotted! A fix, and, critically, a test that proves it works, are in c093e7cf1d